app/utils/constants.ts

```diff
@@ -147,6 +147,27 @@ const PROVIDER_LIST: ProviderInfo[] = [
     getApiKeyLink: 'https://lmstudio.ai/',
     labelForGetApiKey: 'Get LMStudio',
     icon: "i-ph:cloud-arrow-down",
+  },
+  {
+    name: 'NovitaAI',
+    staticModels: [
+      { name: 'qwen/qwen-2.5-72b-instruct', label: 'qwen-2.5-72b-instruct', provider: 'NovitaAI', maxTokenAllowed: 16000 },
+      { name: 'meta-llama/llama-3.1-405b-instruct', label: 'llama-3.1-405b-instruct', provider: 'NovitaAI', maxTokenAllowed: 16384 }
+    ],
+    getApiKeyLink: 'https://novita.ai/settings/api-keys'
+  },
+  {
+    name: 'TogetherAI',
+    staticModels: [
+      { name: 'Qwen/Qwen2.5-72B-Instruct-Turbo', label: 'Qwen2.5-72B-Instruct-Turbo', provider: 'TogetherAI', maxTokenAllowed: 1000 },
+      { name: 'meta-llama/Meta-Llama-3.1-8B-Instruct-Turbo', label: 'Llama 3.1 8B Instruct Turbo', provider: 'TogetherAI', maxTokenAllowed: 1000 },
+    ],
+  },
+  {
+    name: 'Azure',
+    staticModels: [
+      { name: 'gpt-4o', label: 'gpt-4o (resource:key)', provider: 'Azure', maxTokenAllowed: 1000 }
+    ],
```

Comment:
Why max token allowed is 1000? Do you have some documentation link?
Also is there page to get keys from? 
Any instructions on how to test this integration?

Reply:
These are configurable at the azure deployment level.

![image](https://github.com/user-attachments/assets/8eae034d-7856-4ade-9f59-eb37764bf213)

may be user is allowed to input this together with api key?

only way to test is to do a deployment of openai model on azure and input the resource name and key {rn}:{key} format.